homework8/ErikaTatchyn/task5/index.js

```diff
@@ -0,0 +1,47 @@
+const options = {
+  method: "GET",
+  headers: {
+    accept: "application/json",
+    "X-RapidAPI-Key": "85aba5998fmshfcbade3fbc9a714p154dc5jsn7c3562c176a8",
+    "X-RapidAPI-Host": "matchilling-chuck-norris-jokes-v1.p.rapidapi.com",
+  },
+};
+
+console.log(options);
+
+const searchBtn = document.getElementById("search");
+searchBtn.addEventListener("click", fetchData);
+function fetchData() {
+  const input = document.getElementById("text");
+  const value = input.value;
+
+  fetch(
+    `https://matchilling-chuck-norris-jokes-v1.p.rapidapi.com/jokes/search?query=${value}`,
```

Comment:
for `https://matchilling-chuck-norris-jokes-v1.p.rapidapi.com/jokes` u can create separate variable and reuse it. Because u use it more than 1 time